### PR TITLE
Added test case for bug 1258

### PR DESCRIPTION
**Pull request by asgrim**
Test case for https://bugs.xdebug.org/view.php?id=1258

### CODE REVIEW

File: tests/bug01258.phpt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    [14]=>
+    int(1)
+    [16]=>
+    int(1)
+    [17]=>
+    int(1)
+    [20]=>
+    int(1)
+  }
+}
```
Comment (derickr) [inline tests/bug01258.phpt]: I'd prefer the new line at the end.

File: tests/bug01258.phpt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+xdebug_stop_code_coverage();
+var_dump($cc);
+?>
+--EXPECTF--
+bar
+baz
+qux
+default
+array(1) {
+  ["/home/james/workspace/xdebug/tests/bug01258.php"]=>
```
Comment (derickr) [inline tests/bug01258.phpt]: I don't have that directory ;-) I'll fix it before merging until you beat me to it to change it to "%sbug01258.php"

File: tests/bug01258.phpt [hunk 3]
```diff
@@ -0,0 +1,62 @@
+--TEST--
+Test for bug #1258: ensure case statements are covered
+--FILE--
+<?php
+$foo = ['bar', 'baz', 'qux', 'quux'];
+xdebug_start_code_coverage(XDEBUG_CC_UNUSED);
```
Comment (derickr) [inline tests/bug01258.phpt]: Thanks! Does it make sense to add `| XDEBUG_CC_DEAD_CODE` to that bitfield?

**Comment by derickr** [comment]
Merged by hand.

**Comment by asgrim** [comment]
Thanks! :)

**Pull request closed without merging**